spec/rails_helper.rb

```diff
@@ -1,15 +1,13 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
-
 # Prevent database truncation if the environment is production
-abort('The Rails environment is running in production mode!') if Rails.env.production?
-
+abort("The Rails environment is running in production mode!") if Rails.env.production?
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.